GOVERNANCE.md

```diff
@@ -0,0 +1,57 @@
+# `userland-migrations` Team Governance
+
+Meet the current team members of the Node.js userland migrations:
+
+- [@AugustinMauroy](https://github.com/AugustinMauroy) - **Augustin Mauroy** (he/him)
+- [@avivkeller](https://github.com/avivkeller) - **Aviv Keller** (he/him)
+- [@bmuenzenmeyer](https://github.com/bmuenzenmeyer) - **Brian Muenzenmeyer** (he/him)
+- [@JakobJingleheimer](https://github.com/JakobJingleheimer) - **Jacob Smith** (he/him) - Maintainer
+- [@ljharb](https://github.com/ljharb) - **Jordan Harband** (he/him)
+
+## Rights
+
+Any team member can block a pull request or decision. A block can be overturned by a team vote, which requires a quorum of two-thirds of active members and passes with a simple majority (>50%). Pursuant to Node.js policy, the [TSC](https://github.com/nodejs/TSC) may override a team’s vote.
+
+### Active vs inactive membership
+
+### Retaining Membership
+
+Members are expected to engage in project discussions regularly, and members inactive for more than six months may be removed from the active members list. Removed members can be reinstated upon returning to active participation.
+
+## Team nomination
+
+This policy extends the parent Node.js policy.
+
+Current Node.js collaborators are free to join at any time and do not require a nomination.
+
+Current Node.js members who are not collaborators are free to request to join without a nomination or meeting contribution criteria. The request must have no objections after 3 days. All active members must be notified of the request.
+
+Non-members of the Node.js org should meet the following criteria:
+
+- ~2 months of consistent (substantive, non-trivial) contribution/participation.
+  - _Consistent_ means roughly once a week (we all have lives)
+  - _Substantive_ means adds value and does not detract from the issue at hand.
```

Comment:
I think we were gonna go with Aviv's suggestion? (Not sure where the thread went).

Reply:
Wait, I'll go look because I must have missed part of the discussion.

Reply:
I can't follow that link (it goes to a bunch of collapsed comments and idk which one it's supposed to go to).

I'm talking about https://github.com/nodejs/userland-migrations/pull/159#discussion_r2258432084 (the bottom blockquote in Aviv's comment).